lib/jsonapi/fetchdata/parameters/adapter.rb

```diff
@@ -27,7 +27,7 @@ def parse params
 
         def available_parsers
           @available_parsers ||= {
-            'page'     => Parameters::Parsers::Paginate,
+            # 'page'     => Parameters::Parsers::Paginate,
```

Comment:
This page param gave some issues in our project so it should be nice to add the middleware for a scope of routes